.gitignore

```diff
@@ -5,7 +5,6 @@ configs/local.json
 npm-debug.log
 assets-error.log
 .idea/
-.vscode/
```

Comment:
I'm inclined to leave `.vscode` out. I'm not sure what other settings people use and it could conflict

Reply:
Whoops meant to do that in a separate checkin.  I'll remove and do another PR. Microsoft recommends checking it in see https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control.